src/riak_kv_vnode.erl

```diff
@@ -219,7 +219,23 @@ put(Preflist, BKey, Obj, ReqId, StartTime, Options) when is_integer(StartTime) -
 
 put(Preflist, BKey, Obj, ReqId, StartTime, Options, Sender)
   when is_integer(StartTime) ->
-    riak_core_vnode_master:command(Preflist,
+    %% Reminder: command_unreliable() = "even more unreliable"
```

Comment:
unless you have some non-idempotent command, like a counter increment (for example.)